### DISCUSSION TITLE
Where do you get the data for the stat scaling by level from?

### DISCUSSION THREAD

**Discussion by mynian** [simulationcraft/simc#7312] [Q&A]
As the title states, where exactly do you get the data fro the amount of a stat per 1% from?

The data in question can be found in this file here, but do you pull it from somewhere or are you calculating it yourself?
https://github.com/simulationcraft/simc/blob/dragonflight/engine/dbc/generated/sc_scale_data.inc

Specifically I am looking at the conversion rates for the secondary stats like crit, mastery, etc.

**Comment by cremor** [comment]
See https://github.com/simulationcraft/simc/wiki/GameClientData and https://github.com/simulationcraft/simc/wiki/Using-CASC-Extract-and-DBC-Extract
